sw/host/cryptotest/testvectors/data/schemas/kmac_schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/lowRISC/opentitan/master/sw/host/cryptotest/testvectors/data/schemas/kmac_schema.json",
+  "title": "Cryptotest KMAC Tag Verification Test Vector",
+  "description": "A list of testvectors for KMAC Tag Verification testing",
```

Comment:
nit: I'm not sure I'd specify "tag verification" here; the same tests (at least the valid ones) can test tag generation as well, since there's no nondeterminism in KMAC.

Reply:
Good point, does that apply to HMAC as well? That text was just copied from the HMAC parser.

Reply:
Yes, it does -- missed it on the HMAC PR.

Reply:
Ok, I'll update #21151